site/docs/about/guidebook1.0.mdx

```diff
@@ -0,0 +1,55 @@
+---
+name: Guidebook 1.0
+route: /about/guidebook1.0
```

Comment:
I don't know if the "About" section is the best place for this. It can be a bit hard to locate. 

One idea could be to add a new "Tutorials" section and add this there. It could be later be a place for other tutorials for Abstract etc.

Reply:
Good idea!

Reply:
Also I guess the 1.0 in the url isn't good practice. And this might not really be a book, more of a leaflet :D

Reply:
Yeah most likely this gets removed later.